src/autofill.js

```diff
@@ -9,6 +9,10 @@
 
     // chrome is only present in desktop browsers
     if (typeof chrome === 'undefined') {
+        const {isApp} = require('./autofill-utils')
```

Comment:
We only want to inject this if `isApp`. Other platforms use this in a different way. As mentioned, we'll switch soon to a custom form submission detection.